h3-datagram/src/client.rs

```diff
@@ -0,0 +1,51 @@
+//! client API
+
+use std::marker::PhantomData;
+
+use bytes::Buf;
+use h3::{
+    client::Connection,
+    quic::{self, StreamId},
+    Error,
+};
+
+use crate::{
+    datagram::Datagram,
+    datagram_traits::{HandleDatagramsExt, ReadDatagram},
+    quic_traits::{self, RecvDatagramExt, SendDatagramExt},
+};
+
+pub trait ClientHandleDatagramsExt<C, B>
```

Comment:
I think this trait can be removed. `client::Connection`  implements `HandleDatagramsExt` like `server::Connection`